server/src/thirdparty/env.js

```diff
@@ -0,0 +1,7 @@
+const env = {
+  slackClientId: process.env.SLACK_CLIENT_ID,
+  slackClientSecret: process.env.SLACK_CLIENT_SECRET,
+  baseUri: process.env.BASE_URI || 'https://f560d651.ngrok.io',
```

Comment:
`ngrok` url changes everytime you restart it. This pattern is useful when you can fall back to some default value, in this case the fall back will also fall haha, either handle this or take a note of this, as this can cause issues in case there is an error

Reply:
Changed it to localhost